src/summary/documentation/dimensions.resource.tsx

```diff
@@ -0,0 +1,11 @@
+import { openmrsFetch } from "@openmrs/esm-api";
+
+export function getPatientDimensions(
+  patientIdentifer: string,
+  abortController: AbortController
+) {
+  return openmrsFetch(
+    `/ws/fhir/Observation?patient.identifier=${patientIdentifer}`,
```

Comment:
Let's try to retrieve this via `fhir.read()` instead of `openmrsFetch`. See https://wiki.openmrs.org/display/projects/openmrs-esm-api and https://github.com/FHIR/fhir.js

Reply:
We have an example of use in our project?

Reply:
No - here are the docs: https://github.com/FHIR/fhir.js